flexmeasures/ui/templates/defaults.jinja

```diff
@@ -34,14 +34,17 @@
 {% endfor %}
 
 
-{% set admin_view = True if current_user.is_authenticated and (current_user.has_role('admin') or FLEXMEASURES_MODE == "demo") else False %}
-{% do navigation_bar.append(('accounts', 'accounts', 'Accounts', '', 'cubes')) if admin_view %}
-{% do navigation_bar.append(('assets', 'assets', 'Assets', '', 'list-ul')) if admin_view %}
-{% do navigation_bar.append(('users', 'users', 'Users', '', 'users')) if admin_view %}
-{% do navigation_bar.append(('tasks', 'tasks', 'Tasks', '', 'tasks')) if admin_view %}
-
-{% set non_admin_view = True if current_user.is_authenticated and not (current_user.has_role('admin') or FLEXMEASURES_MODE == "demo") else False %}
-{% do navigation_bar.append(("accounts/{}".format(current_user.account.id), 'accounts', 'My Account', '', 'cubes')) if non_admin_view %}
+{% set view = "admin" if current_user.is_authenticated and (current_user.has_role('admin') or FLEXMEASURES_MODE == "demo") %}
+{% do navigation_bar.append(('accounts', 'accounts', 'Accounts', '', 'cubes')) if view == "admin" %}
+{% do navigation_bar.append(('assets', 'assets', 'Assets', '', 'list-ul')) if view == "admin" %}
+{% do navigation_bar.append(('users', 'users', 'Users', '', 'users')) if view == "admin" %}
+{% do navigation_bar.append(('tasks', 'tasks', 'Tasks', '', 'tasks')) if view == "admin" %}
+
+{% set view = "non_admin" if current_user.is_authenticated and not (current_user.has_role('admin') or FLEXMEASURES_MODE == "demo" or current_user.has_role('customer-manager')) %}
```

Comment:
Difficult to read. Let's stay as atomic and I would use separate flags, not just `view`.
Maybe `is_admin` and `is_consultant` (one can be both).

I guess `non_admin` is not a good term here - and I would not use it.

We already have  `admin` and `consultant` -  "My account" should be shown if the user is authenticated, is neither admin nor consultant. And apparently when not in demo mode either?

Reply:
Refactored the code. The original was quite confusing as well, I think the current version is the most clear and can be understood and changed.